backend/src/main/java/gov/cdc/usds/simplereport/db/model/TestEvent.java

```diff
@@ -0,0 +1,57 @@
+package gov.cdc.usds.simplereport.db.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+
+import org.hibernate.annotations.Immutable;
+import org.hibernate.annotations.Type;
+
+import gov.cdc.usds.simplereport.db.model.auxiliary.TestResult;
+
+@Entity
+@Immutable
+public class TestEvent extends AuditedEntity {
+
+	@Column
+	@Type(type = "pg_enum")
+	@Enumerated(EnumType.STRING)
+	private TestResult result;
+	@ManyToOne
+	@JoinColumn(name = "device_type_id")
+	private DeviceType deviceType;
+	@Column
+	@Type(type = "jsonb")
+	private Person patientData;
```

Comment:
I think we probably want to make this some kind of a non-persisted model instead of a weird disconnected version of the actual `Person` entity. But this does get the data stored...